src/Elcodi/Component/Cart/Entity/CartLine.php

```diff
@@ -142,4 +152,62 @@ public function getWeight()
             ->getPurchasable()
             ->getWeight();
     }
+
+    /**
+     * Set taxPercentage
+     *
+     * @param float $taxPercentage
+     *
+     * @return $this Self object
+     */
+    public function setTaxPercentage($taxPercentage)
+    {
+        $this->taxPercentage = $taxPercentage;
+
+        return $this;
+    }
+
+    /**
+     * Get taxPercentage
+     *
+     * @return float
+     */
+    public function getTaxPercentage()
+    {
+        return $this->taxPercentage;
+    }
+
+    /**
+     * Get Taxed Amount
+     *
+     * @return MoneyInterface
+     */
+    public function getTaxedAmount()
+    {
+        return \Elcodi\Component\Currency\Entity\Money::create(
+            $this->amount,
+            $this->currency
+        )->add( $this->getTaxAmount() );
+    }
+
+    /**
+     * Get Tax Amount
+     *
+     * @return MoneyInterface
+     */
+    public function getTaxAmount()
+    {
+        if( isset( $this->taxPercentage ) )
```

Comment:
Please, follow the php-cs format.
* php bin/php-cs-fixer fix
* php bin/php-formatter f:h:f src
* php bin/php-formatter f:u:s src